### PR TITLE
translate index.en

### PR DESCRIPTION
**Pull request by hp0632**
fix the index_en.rst file

### CODE REVIEW

**Comment by shanyi15** [comment]
close via https://github.com/PaddlePaddle/Paddle/pull/9785

File: doc/v2/build_and_install/index_en.rst [hunk 1]
```diff
@@ -1,32 +1,57 @@
-Install and Build
-=================
+install and compile
```
Comment (abhinavarora) [inline doc/v2/build_and_install/index_en.rst]: compile -> **Compile**

File: doc/v2/build_and_install/index_en.rst [hunk 2]
```diff
@@ -1,32 +1,57 @@
...
+==========
 
 .. _install_steps:
 
-Install Steps
-++++++++
+PaddlePaddle provides various ways of installation for many different users
```
Comment (abhinavarora) [inline doc/v2/build_and_install/index_en.rst]: ways -> **methods**

File: doc/v2/build_and_install/index_en.rst [hunk 3]
```diff
@@ -1,32 +1,57 @@
...
 
-You can choose either pip or Docker to complete your install:
+focus on deep learning model development
```
Comment (abhinavarora) [inline doc/v2/build_and_install/index_en.rst]: focus on deep learning model development ->  Focus on Deep Learning Model Development

File: doc/v2/build_and_install/index_en.rst [hunk 4]
```diff
@@ -1,32 +1,57 @@
...
-   :maxdepth: 1
+	:maxdepth: 1
 
-   pip_install_en.rst
-   docker_install_en.rst
+	pip_install_cn.rst
 
-Build from Source
------------------
+this is the convenient way to install it , please choose the right installation package with mochine configure and system。
```
Comment (abhinavarora) [inline doc/v2/build_and_install/index_en.rst]: this -> This

Comment (abhinavarora) [inline doc/v2/build_and_install/index_en.rst]: the convenient way -> the **most** convenient way

File: doc/v2/build_and_install/index_en.rst [hunk 5]
```diff
@@ -1,32 +1,57 @@
...
+
+follow the bottom frame
```
Comment (abhinavarora) [inline doc/v2/build_and_install/index_en.rst]: follow -> **Follow**

File: doc/v2/build_and_install/index_en.rst [hunk 6]
```diff
@@ -1,32 +1,57 @@
...
+----------
+
+PaddlePaddle provides the installation ways of Docker, please follow the tutorial:
```
Comment (abhinavarora) [inline doc/v2/build_and_install/index_en.rst]: This sentence should be:

PaddlePaddle provides installation methods using Docker. Please follow the tutorial:

File: doc/v2/build_and_install/index_en.rst [hunk 7]
```diff
@@ -1,32 +1,57 @@
...
+
+.. toctree::
+	:maxdepth: 1
+
+	docker_install_cn.rst
+
+we recommend running PaddlePaddle in docker , this way has more superiority ：
```
Comment (abhinavarora) [inline doc/v2/build_and_install/index_en.rst]: we ->We
docker -> Docker

Comment (abhinavarora) [reply doc/v2/build_and_install/index_en.rst]: This sentence shouldbe 

We recommend running PaddlePaddle using Docker.

Comment (abhinavarora) [reply doc/v2/build_and_install/index_en.rst]: this way has more superiority  -> This method has the following advantages:

File: doc/v2/build_and_install/index_en.rst [hunk 8]
```diff
@@ -1,32 +1,57 @@
...
 
-..  warning::
+- don't need the third dependent of installing
```
Comment (abhinavarora) [inline doc/v2/build_and_install/index_en.rst]: don't need the third dependent of installing ->Does not require the installation of third-party dependencies.

**Pull request closed without merging**